Stellar-ledger-entries-fee.x

```diff
@@ -21,36 +22,49 @@ enum FeeType
     ATOMIC_SWAP_PURCHASE_FEE = 9
 };
 
+//: (not used yet)`EmissionFeeType` is a subtype of the Fee used for issuance
 enum EmissionFeeType
 {
-	PRIMARY_MARKET = 1,
-	SECONDARY_MARKET = 2
+    PRIMARY_MARKET = 1,
+    SECONDARY_MARKET = 2
 };
 
+// `PaymentFeeType` is a subtype of the Fee used for payments
 enum PaymentFeeType
 {
     OUTGOING = 1,
     INCOMING = 2
 };
 
+// `FeeEntry` represents the fee structure with the corresponding details
```

Comment:
FeeEntry is used in system configuration to set fees for different operations, assets, _amount ranges_(Muh, rephrase)